tests/test_cli.py

```diff
@@ -237,6 +278,47 @@ def test_mlflow_gc_not_deleted_run(file_store):
     assert len(runs) == 1
 
 
+def test_mlflow_gc_file_store_older_than(file_store):
+    store = file_store[0]
+    run = _create_run_in_store(store)
+    store.delete_run(run.info.run_uuid)
+    with pytest.raises(subprocess.CalledProcessError, match=r".+") as exp:
+        subprocess.run(
+            [
+                "mlflow",
+                "gc",
+                "--backend-store-uri",
+                file_store[1],
+                "--older-than",
+                "10d10h10m10s",
+                "--run-ids",
+                run.info.run_uuid,
+            ],
+            check=True,
+            capture_output=True,
+            text=True,
+        )
+        assert "is not older than the required age" in exp.value.stderr
```

Comment:
This assert statement is ignored because `subprocess.run` throws.

```suggestion
    assert "is not older than the required age" in exp.value.stderr
```


https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest.raises for more information.

<img width="881" alt="image" src="https://user-images.githubusercontent.com/17039389/182647098-07308561-fc25-406f-b7df-0986808a77eb.png">

Reply:
Thanks for the information. I have changed it such that it runs the assert statement and passed the pytest for this file!